docs/guides/snowflake_integration.md

```diff
@@ -431,3 +434,75 @@ The following table identifies each item in the response. The result includes an
 |  3 | test2@uidapi.com   | NULL                                         | NULL       | NULL                    | NULL                    |
 +----+--------------------+----------------------------------------------+------------+-------------------------+-------------------------+
 ```
+
+## Migration Guide
+
+If you are using the `FN_T_UID2_IDENTITY_MAP_EMAIL` and `FN_T_UID2_IDENTITY_MAP_EMAIL_HASH` functions, it's best to migrate to the `FN_T_UID2_IDENTITY_MAP` function as soon as possible. This function does everything that the other two functions do, and has other built-in improvements.
+
+Advantages of the `FN_T_UID2_IDENTITY_MAP` function:
+
+- It supports mapping both phone numbers and hashed phone numbers.
+- It supports user opt-out.
+- It adds a new column, `UNMAPPED`. In any scenario where the DII cannot be mapped to a UID2 for any reason, this column includes information about the reason. For details, see [Values for the UNMAPPED Column](#values-for-the-unmapped-column)
+
+This section includes the following information to help you upgrade to the new function:
+
+- [Changing Existing Code](#changing-existing-code) 
+- [Using the Values for the UNMAPPED Column](#using-the-values-for-the-unmapped-column) 
+
+### Changing Existing Code
+
+The code snippets in this section are before/after examples of how the earlier functions might be implemented, and how you could update to use the new function.
+
+#### Example for mapping unhashed emails
+
+Before:
+
+```
+FN_T_UID2_IDENTITY_MAP_EMAIL(EMAIL)
+```
+
+After:
+
+```
+FN_T_UID2_IDENTITY_MAP(EMAIL, 'email')
+```
+
+#### Example for mapping unhashed emails
+
+Before:
+
+```
+FN_T_UID2_IDENTITY_MAP_EMAIL_HASH(EMAIL_HASH)
+```
+
+After:
+
+```
+FN_T_UID2_IDENTITY_MAP(EMAIL_HASH, 'email_hash')
+```
+
+### Using the Values for the UNMAPPED Column
+When you have the new function implemented, you can check the `UNMAPPED` column returned by the `FN_T_UID2_IDENTITY_MAP`. If any DII could not be mapped to a UIS2, this column gives the reason.
+
+For details about the values and their explanations, see [Values for the UNMAPPED Column](#values-for-the-unmapped-column).
+
+{**MC/GWH: we already have this info in the [Values for the UNMAPPED Column](#values-for-the-unmapped-column) section. Can I just point to that rather than having it twice? Please look -- I reworked the explanations in that section a little, and I added the detail about the values for `INVALID INPUT TYPE` into the table above.**}
```

Comment:
Yes, I agree it's better to have that information in a single place.

Reply:
TY. See update.